app/src/views/QuestionView.vue

```diff
@@ -4,77 +4,74 @@
+    <span id = "qText" v-html="question.text"></span></div>
     <!-- Old code with best answer
     <div class = "bestAnswer" v-if="answers.length>0"><h2>Most highly rated answer:</h2><VoteChoice v-bind:id="answers[0].id" v-bind:type="post"/>{{answers[0].text}}
     <div class = "comment" v-for="comment in answers[0].comments"><VoteChoice v-bind:id="comment.id" v-bind:type="comment"/>{{comment.text}}</div></div>
     <div class = "answerBox" v-for="answer in answers" v-if="answer != answers[0]"><VoteChoice v-bind:id="answer.id" v-bind:type="post"/><p>{{answer.text}}</p>
     <div class = "comment" v-for="comment in answer.comments">
     <VoteChoice  v-bind:id="comment.id" v-bind:type="comment"/>{{comment.text}}</div>-->
-
     <div class = "sorting">
         Sort by:
-        <select v-model="sort" placeholder>
+        <select>
             <option selected>Time</option>
             <option>Score</option>
         </select>
     </div>
 
     <!-- v-bind:canVote="true" is temporary -->
-    <div class = "answerBox" v-for="answer in answers" v-bind:key="answer.id"><VoteChoice v-bind:id="answer.id" v-bind:type="post" v-bind:score="answer.score" v-bind:canVote="true"/><p>{{answer.text}}</p>
+    <div class = "answerBox" v-for="answer in answers" v-bind:key="answer.id" >
+    <VoteChoice v-bind:id="answer.id" v-bind:type="'post'" v-bind:score="answer.score" v-bind:canVote="true"/><span v-html="answer.text"></span>
     <div class = "comment" v-for="comment in answer.comments" v-bind:key="comment.id">
     <VoteChoice  v-bind:id="comment.id" v-bind:type="comment"/>{{comment.text}}</div>
-    <CommentButton v-bind:id='answer.id'/>
+    <div class = "commentButton"><button type="button" @click = "postComment(answer)">Comment</button></div>
     </div>
     <div class = "post_section">
-    <form>
-      <textarea v-model="message" style = "width: 80%; height: 118px;" placeholder="Insert text here"></textarea>
+    <form @submit.prevent="addAnswer" >
+      <textarea v-model="text" name = "text" style = "width: 80%; height: 118px;" placeholder="Insert text here"></textarea>
       <br>
-      <input type = "submit" value = "Post" v-on:click = "submitPost">
+      <input type = "submit" value = "Post" >
     </form>
     </div>
   </div>
 </template>
 
 <script>
+import apiService from '../services/ApiService.js'
 import VoteChoice from '../components/VoteChoice.vue'
-import CommentButton from '../components/CommentButton.vue'
 export default {
   name: 'QuestionView',
+  methods: {
+    getData: function () {
+      return apiService.get('questions/'+this.$route.params.id)
+      },
+    addAnswer() {
+      apiService.post('questions/'+this.$route.params.id+'/answers', {text: this.text}).then(res => this.answers = [...this.answers, res])
+      this.text = ''
+    },
+    postComment: function (answer){
+            apiService.post('answers/'+answer.id+'/comments', {text: this.text}).then(res=>[...answer.comments, res])
```

Comment:
small tip, in javascript you can use backticks (`) and pass the value inside like this: `answers/${answer.id}/comments`